packages/draft-js-export-html/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "draft-js-export-html",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
`1.2.1`?

Reply:
nvm, I think I see

Reply:
yeah we didn't actually update this but the repo had changes that weren't published for this package.  We could remove this change all together 🤷‍♂️